### PR TITLE
[510] SPESS PARALLAX, but for the 510 branch.

### PR DESCRIPTION
**Pull request by monster860**
It's #3734 with a few extra commits to make it, well, 510-compatible.

### CODE REVIEW

File: paradise.dme [hunk 1]
```diff
@@ -5,6 +5,90 @@
 // END_INTERNALS
 // BEGIN_FILE_DIR
 #define FILE_DIR .
+#define FILE_DIR "html"
```
Comment (Fox-McCloud) [inline paradise.dme]: you forgot to untick this....

**Comment by monster860** [comment]
Craaap.

File: code/game/atoms_movable.dm [hunk 2]
```diff
@@ -88,6 +88,9 @@
 			if(loc == newloc) //Remove this check and people can accelerate. Not opening that can of worms just yet.
 				newtonian_move(last_move)
 
+	for(var/mob/M in contents)
+		M.Move(newloc,direct)
```
Comment (tigercat2000) [inline code/game/atoms_movable.dm]: Did I forget to comment on this? mob/Move() being called while they are inside containers is, to say the least, completely fucking insane. Clowns making boot noises while in lockers or flying through space in a spacepod.

Snowflaking 
```dm
for(var/mob/M in contents)
    if(M.hud_used)
        M.hud_used.update_parallax()
```
is actually better.

**Comment by monster860** [comment]
@tigercat2000, is there anything I have unfixed?

**Comment by TheDZD** [comment]
Probably shouldn't have a changelog, since this isn't going to the server yet.

**Comment by monster860** [comment]
So should I add a changelog as a .yml file then?

**Comment by TheDZD** [comment]
No changelog at all should be fine, the PR to merge the 510 branch into master will need a fairly large changelog by itself.

**Comment by monster860** [comment]
Okay this should be fine then.

**Comment by monster860** [comment]
Soooo merge when?

**Comment by Dumbdumn5** [comment]
Ello?

**Comment by Fox-McCloud** [comment]
Would really prefer 510 be merged to master first and all the isuses/bugs resolved with that before we consider this.

**Comment by monster860** [comment]
Shiiiiiit.

**Comment by Spacemanspark** [comment]
Holy fuck

**Comment by TheDZD** [comment]
That's a lot of commits.

**Pull request closed without merging**